src/aspire/volume/symmetry_groups.py

```diff
@@ -132,43 +124,40 @@ def generate_rotations(self):
         :return: Rotation object containing the Cn symmetry group and the identity.
         """
         angles = 2 * np.pi * np.arange(self.order) / self.order
-        return Rotation.about_axis("z", angles, dtype=self.dtype)
+        return Rotation.about_axis("z", angles, dtype=np.float64)
```

Comment:
question: is this computing in doubles and casting to doubles, or computing in the type of `angles` and then casting to doubles regardless.

I think angles would be doubles as written, but this might be fragile if that line changed.

Reply:
This is computing in the type of `angles` and then casting to doubles regardless.

You are correct, `angles` is doubles as written. I'll remove passing this dtype.

I initially ran into a problem when angles was `np.pi` in the lines below. It was always returning singles, but I realize now that was happening because our dtype check in `about_axis` is using `getattr(angles, "dtype", np.float32)` and `np.pi` does not have a dtype attribute.

Reply:
I ended up reverting to keep the `self.dtype` attribute. In the case of using `about_axis` (here and below) I apply the dtype to the angles prior to passing into `about_axis`. This way the dtype is inferred from angles and the computation is done using that dtype as well.